planner/cascades/integration_test.go

```diff
@@ -41,20 +40,42 @@ func (s *testIntegrationSuite) SetUpSuite(c *C) {
 	var err error
 	s.store, err = newStoreWithBootstrap()
 	c.Assert(err, IsNil)
-	s.tk = testkit.NewTestKitWithInit(c, s.store)
-	s.tk.MustExec("set session tidb_enable_cascades_planner = 1")
 }
 
 func (s *testIntegrationSuite) TearDownSuite(c *C) {
 	s.store.Close()
 }
 
 func (s *testIntegrationSuite) TestSimpleProjDual(c *C) {
-	s.tk.MustQuery("explain select 1").Check(testkit.Rows(
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec("set session tidb_enable_cascades_planner = 1")
+	tk.MustQuery("explain select 1").Check(testkit.Rows(
 		"Projection_3 1.00 root 1",
 		"└─TableDual_4 1.00 root rows:1",
 	))
-	s.tk.MustQuery("select 1").Check(testkit.Rows(
+	tk.MustQuery("select 1").Check(testkit.Rows(
 		"1",
 	))
 }
+
+func (s *testIntegrationSuite) TestPKIsHandleRangeScan(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(a int primary key, b int)")
+	tk.MustExec("insert into t values(1,2),(3,4)")
+	tk.MustExec("set session tidb_enable_cascades_planner = 1")
+	tk.MustQuery("explain select b from t where a > 1").Check(testkit.Rows(
```

Comment:
How about to:
1. more this test to the `explaintest`
2. add a function for `tk` named `MustTableRead(sql)`, like `MustIndexLookup()`, check whether the execution plan contains `TableReader`, execute the sql and return the result.